wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -116,6 +120,27 @@ public void testGetEnabledJobsByType() throws IOException {
     Files.deleteIfExists(path2);
   }
 
+  @Test
+  public void testMultipleLinesSameProperties() throws IOException, ConfigurationException {
+    String dir = "/tmp/";
+    String propertyStr =
+        "var1=foo\n" + "var1=foo\n" + "var1=foo\n" + "var2=foo\n" + "var2=foo\n" + "var3=foo\n" + "var1=foo\n"
+            + "var4=foo\n";
+    String propertyStr1 = "var5=foo,bar,test,test1\n" + "var5=test2\n";
```

Comment:
Just append this to propertyStr directly instead of creating another variable?

Reply:
Sure